src/Jira.js

```diff
@@ -86,12 +74,28 @@ export default class Jira {
       promises.push(
         this.getJiraIssue(id)
         .then((ticket) => {
-          if (includeTicket(ticket)) {
+          if (this.includeTicket(ticket)) {
             log.tickets.push(ticket);
+            return ticket;
           }
         })
         .catch((err) => {
-          console.log(`Ticket ${id} not found: ${err}`);
+          console.log('Ticket not found', id);
+        })
+        .then((ticket) => {
+
+          // Add release version
+          if (ticket && releaseVersion) {
+            ticket.fields.fixVersions.push({name: releaseVersion});
+            return this.jira.updateIssue(ticket.id, {
+              fields: {
+                'fixVersions': ticket.fields.fixVersions
+              }
+            });
+          }
+        })
+        .catch((err) => {
+          console.log('Failed to attach release version to ticket: ', id, ' ',  err.error.errors);
```

Comment:
Generally `console.log` automatically puts space between the concatenated arguments.